module/os/linux/zfs/zfs_vfsops.c

```diff
@@ -1264,14 +1264,22 @@ zfs_prune(struct super_block *sb, unsigned long nr_to_scan, int *objects)
 	defined(SHRINK_CONTROL_HAS_NID) && \
 	defined(SHRINKER_NUMA_AWARE)
 	if (shrinker->flags & SHRINKER_NUMA_AWARE) {
+		long tc = 1;
+		for_each_online_node(sc.nid) {
+			long c = shrinker->count_objects(shrinker, &sc);
+			if (c  == 0 || c == SHRINK_EMPTY)
```

Comment:
Do we need to check for `SHRINK_EMPTY` since I do not see it being returned from `arc_shrinker_count()`.

Reply:
That is what kernel does, I've copied form there.

Reply:
PS: And we are not calling arc_shrinker_count() here, but a kernel's superblock one.

Reply:
Thanks for the clarification.